code/cfile/cfile.cpp

```diff
@@ -235,6 +236,9 @@ int cfile_init(const char *exe_dir, const char *cdrom_dir)
 		Cfile_block_list[i].type = CFILE_BLOCK_UNUSED;
 	}
 
+	// Init to an empty string to avoid having to test for NULL everywhere
+	Pathtypes[CF_TYPE_TEMP_SUBDIR_LOOKUP].path = vm_strdup("");
```

Comment:
This memory is leaked. This can be seen in the valgrind results of the unit tests.